lib/src/state_manager/rx/rx_impl.dart

```diff
@@ -33,15 +50,24 @@ class _RxImpl<T> implements RxInterface<T> {
 
   String get string => value.toString();
 
-  close() {
-    _subscriptions.forEach((observable, subscription) {
-      subscription.cancel();
-    });
+  @override
+  String toString() => value.toString();
+
+  /// This equality override works for _RxImpl instances and the internal values.
+  bool operator ==(o) {
```

Comment:
just to be sure i understood the code right: before, i could not compare Rx for equality and now i can?

so
```dart
final user = User().obs;
final user2 = User().obs;

assert(user == user2); // this would return false before and now it returns true? just to be sure

Reply:
Only if u override inside User() the == and/or hashCode... but yeah it compares the inner "native" value type instead of the Rx

Reply:
So we don't need to do `user.value == user2.value` right?

Reply:
@ghprod Exactly. It compares against native type (`User()` or int, bool, etc), or the value of another Rx (`Rx<User>::value`)... so it behaves more like a "proxy" of the actual value. That's why is lacking the type in the implementation (as a **self reminder**, should be dynamic :P ).